rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/90_sparse_vector.yml

```diff
@@ -1,9 +1,9 @@
 ---
-"Indexing and searching sparse vectors in >=8.11":
+"Indexing and searching sparse vectors in >=8.15":
 
   - requires:
-      cluster_features: ["gte_v8.11.0"]
-      reason: "sparse_vector field type reintroduced in 8.11"
+      cluster_features: ["gte_v8.15.0"]
```

Comment:
we should instead add a new "Indexing and searching multi-valued sparse vectors in >=8.15"

And ensure this test still covers the single sparse vector test case in the appropriate versions.

Reply:
Ah that makes some sense; i'll fix this.